tests/regression/rhbz-1584461/task.yaml

```diff
@@ -19,7 +19,8 @@ restore: |
     rm -f ./in-snap ./on-host
 
 execute: |
-    # on Fedora, /etc/nsswitch.conf is a symlink to /etc/authselect/nsswitch.conf
+    # on Fedora 30 and 31, /etc/nsswitch.conf is a symlink to /etc/authselect/nsswitch.conf
+    # on Fedora 32, /etc/nsswitch.conf is not a symlink to /etc/authselect/nsswitch.conf
```

Comment:
Looks like F32 is using a non authselect setup by default now. Perhaps it's still useful to be able to run the test with such setup though. For that you'd need to run the following in image prepare:
```
dnf install autselect -y
autselect select minimal --force
```

Reply:
@bboozzoo thanks for the review, I already did that but the tests still failing.